NuGet.config

```diff
@@ -2,6 +2,7 @@
 <configuration>
   <packageSources>
     <clear />
+    <add key="VSPackages" value="https://devdiv.pkgs.visualstudio.com/DefaultCollection/_packaging/VS/nuget/v3/index.json" />
```

Comment:
You might want to try the public vs-impl feed: `https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl%40Local/nuget/v3/index.json`

If the packages you need aren't on that feed, maybe the owners should consider putting them there.

Reply:
Nice suggestion. Unfortunately they're not available there either 😢. @milopezc and friends are working towards a resolution. I doubt we'll be the only ones to hit this in the near future

Reply:
Right.  Hence the proposal to get those packages on that feed - all the DevDiv feeds are for internal DevDiv AzDO use only, this is the feed to use to share VS pre-release packages with OSS projects.